src/assets/scss/app.scss

```diff
@@ -64,7 +64,12 @@
 @include motion-ui-animations;
 
 $message-new: $white;
+$message-new-text: #404040;
```

Comment:
Bod k zamyšlení, co kdybych tvoji komponentu chtěl naimportit do svého projektu, ale úplně v jiných barvách ? Nebylo by fajn, kdybych ti ty proměnný mohl přetížit - co třeba použít výchozí vlastnost proměnné v sassu (!default)... :)
A ještě krasnější, kdybys mi to dal do samostatného souboru., nebo nějak uhladil a já bych pak věděl vlastně co si můžu  kastomizovat.. třeba nějak takhle:
https://github.com/dimsemenov/Magnific-Popup/blob/master/src/css/_settings.scss

Reply:
Zatim jsem to pridal do souboru spolecne s celou componentou. Momentalne tam neni moc veci na meneni

Reply:
to nevadí, jde pouze o demonstraci návrhu..